plugins/ReverbSC/dcblock.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * DCblock
+ *
+ * This code has been extracted from the Csound opcode "dcblock".
+ * It has been modified to work as a Soundpipe module.
+ *
+ * Original Author(s): Perry R. Cook
+ * Year: 1995
+ * Location: Opcodes/biquad.c
+ *
+ */
```

Comment:
Soundpipe was switched fro LGPL to MIT license here:
https://github.com/PaulBatchelor/Soundpipe/blob/master/LICENSE